torch/export/_trace.py

```diff
@@ -707,13 +707,18 @@ def _maybe_fixup_gm_and_output_node_meta(old_gm, new_gm):
         + len(graph_signature.buffers)
         + len(graph_signature.input_tokens)
     )
+    fake_mode = detect_fake_mode(flat_fake_args)
     for node in gm.graph.nodes:
         if node.op == "placeholder":
             if index >= total_non_user_inputs:
                 user_arg = flat_fake_args[index - total_non_user_inputs]
                 if not isinstance(user_arg, torch.Tensor):
                     node.meta["val"] = user_arg
             index += 1
+        if node.op == "get_attr" and "val" not in node.meta:
+            val = getattr(gm, node.target)
+            if val in constant_attrs and isinstance(val, torch.Tensor):
+                node.meta["val"] = fake_mode.from_tensor(val, static_shapes=True)
```

Comment:
I'm confused why this is not getting handled [here](https://github.com/pytorch/pytorch/blob/56da78f8faba7a7a9130caddd11c47bad23969b4/torch/_export/passes/lift_constants_pass.py#L229-L237)?

Reply:
That code is in the lift constant pass, whereas we need to generate a graph signature before that, and the missing meta val causes graph signature generation to crash.